### PR TITLE
Add --snapshot-id option to update in disco_snapshots.py

### PR DESCRIPTION
**Pull request by pchand-amplify**
Adding the option `--snapshot-id` to update autoscaling group when we already have the snapshot_id in hand.

- [x] code
- [ ] test with a hostclass

### CODE REVIEW

File: disco_aws_automation/disco_storage.py [hunk 1]
```diff
@@ -362,3 +362,13 @@ def take_snapshot(self, volume_id, snapshot_tags=None):
...
+
+    def get_snapshot_from_id(self, snapshot_id):
+        """
+        Lists all EBS snapshots associated with a hostclass, sorted by hostclass name and start_time
+
+        :param hostclasses if not None, restrict results to specific hostclasses
+        """
+        return throttled_call(self.connection.get_all_snapshots,
+                              snapshot_ids=[snapshot_id],
+                              filters={'tag:env': self.environment_name})[0]
```
Comment (jconstance-amplify) [inline disco_aws_automation/disco_storage.py]: Is the filter actually necessary?

File: disco_aws_automation/disco_storage.py [hunk 2]
```diff
@@ -362,3 +362,13 @@ def take_snapshot(self, volume_id, snapshot_tags=None):
         throttled_call(snapshot.add_tags, tags=tags)
 
         return snapshot.id
+
+    def get_snapshot_from_id(self, snapshot_id):
+        """
+        Lists all EBS snapshots associated with a hostclass, sorted by hostclass name and start_time
+
+        :param hostclasses if not None, restrict results to specific hostclasses
```
Comment (jconstance-amplify) [inline disco_aws_automation/disco_storage.py]: This comment seems out of date.

File: bin/disco_snapshot.py [hunk 3]
```diff
@@ -125,7 +126,10 @@ def run():
         for snapshot_id in args.snapshots:
             aws.disco_storage.delete_snapshot(snapshot_id)
     elif args.mode == "update":
-        snapshot = aws.disco_storage.get_latest_snapshot(args.hostclass)
+        if args.snapshot_id is None:
```
Comment (sshrestha-amplify) [inline bin/disco_snapshot.py]: nitpick : 
`if args.snapshot_id:
    use snapshot id
else:
    get latest`

File: disco_aws_automation/disco_storage.py (hunk 1 above)
Comment (pchand-amplify) [inline disco_aws_automation/disco_storage.py]: Not really.

File: disco_aws_automation/disco_storage.py (hunk 2 above)
Comment (pchand-amplify) [inline disco_aws_automation/disco_storage.py]: True. Thought I had fixed it. :'(

File: bin/disco_snapshot.py (hunk 3 above)
Comment (pchand-amplify) [inline bin/disco_snapshot.py]: 👍

**Comment by sshrestha-amplify** [comment]
Other than nitpick and John's comment about old comments -> LGTM

**Pull request merged**